src/BrowserKitDriver.php

```diff
@@ -413,7 +413,24 @@ public function getValue($xpath)
      */
     public function setValue($xpath, $value)
     {
-        $this->getFormField($xpath)->setValue($value);
+        $formField = $this->getFormField($xpath);
+
+        try {
```

Comment:
Ok, it set this value by BrowserKit without exception. It's ok for multiple select as well